bin/debug/extract_timeline_for_day_range_and_user.py

```diff
@@ -19,82 +19,34 @@
 import arrow
 import argparse
 
-import emission.core.wrapper.user as ecwu
-import emission.storage.timeseries.abstract_timeseries as esta
 import emission.storage.timeseries.timequery as estt
+import emission.storage.timeseries.abstract_timeseries as esta
```

Comment:
can we avoid irrelevant changes like this (remove a line and add it back)? It complicates the change history for no good reason.

Reply:
Fixed